server/commands.lua

```diff
@@ -28,6 +28,44 @@ lib.addCommand('blips', {
     TriggerClientEvent('qbx_admin:client:blips', source)
 end)
 
+lib.addCommand('admincar', {
+    help = 'Buy Vehicle',
+    restricted = config.saveVeh,
+}, function(source)
+    local vehicle = GetVehiclePedIsIn(GetPlayerPed(source), false)
+    if not vehicle then
```

Comment:
```suggestion
    if vehicle == 0 then
```

Vehicle will never be nil/false, only 0 or a number greater like 421352